src/pygeoprocessing/__init__.py

```diff
@@ -7,6 +7,7 @@
 import types
 
 from . import geoprocessing
+from .geoprocessing import ReclassificationMissingValuesError
```

Comment:
Could you add `ReclassificationMissingValuesError`  to `__all__` as well here?  Doing so will enable `from pygeoprocessing import *` to also expose `ReclassificationMissingValueError`.

Reply:
Added to `__all__` here: https://github.com/natcap/pygeoprocessing/pull/106/commits/5eecc74c5c71675b465c97261efedacbe8ce3f8e